dmon.c

```diff
@@ -13,6 +13,7 @@
 #include "task.h"
 #include "util.h"
 #include <assert.h>
+#include <sys/sysinfo.h>
```

Comment:
I would also add an `#if defined(__UCLIBC__)` conditional before including `<sys/sysinfo.h>` because the header is Linux-specific, and DMon works on other systems as well. Or, better, add the `#include` below (see my other comment).

Reply:
Followed your other comment.